package.json

```diff
@@ -105,6 +106,7 @@
     "gitbook-cli": "^2.3.0",
     "glob": "^6.0.4",
     "isparta": "^4.0.0",
+    "jest": "^15.1.1",
     "mocha": "^2.2.5",
```

Comment:
Thanks @ellbee 

I removed the dependencies from package.json and updated `test:cov` to use jest